locallib.php

```diff
@@ -2717,6 +2730,13 @@ function bigbluebuttonbn_settings_record(&$renderer) {
             'recording_editable',
             $renderer->render_group_element_checkbox('recording_editable', 1)
         );
+        // If opencast plugin is installed.
```

Comment:
Please align this comment with the other comments in the patch.
```suggestion
        // If block_opencast is installed.
```